sockeye/arguments.py

```diff
@@ -215,6 +230,28 @@ def add_model_parameters(params):
                               help="Add residual connections to stacked RNNs if --rnn-num-layers > 3. "
                                    "(see Wu ETAL'16). Default: %(default)s.")
 
+    # transformer arguments
+    model_params.add_argument('--transformer-model-size',
+                              type=int_greater_or_equal(1),
+                              default=512,
+                              help='Size of all layers and embeddings when using transformer. Default: %(default)s.')
+    model_params.add_argument('--transformer-num-layers',
```

Comment:
That would be a possibility. But all of our arguments are currently 'tied' to the type of model. we have attention-num-hidden, rnn-num-hidden and all the conv-seq-embed arguments. On the other hand, the number of layers tends to be a fairly general concept across models, so I will make the change.